### ISSUE TITLE
[Request] Option for loading /proc/config.gz

### ISSUE THREAD

**Issue by shelterx** [CachyOS/kernel-manager#22]
We have "Use modpobed-db" as an option which is nice but I would really like to see "Use the running kernel config" or something similar, so it loads /proc/config.gz.
I guess there's an issue with implementing something like this because it will override every setting in KM but as long as your clear with that, it should work.

The way I do it now is I enable menuconfig, start the build, then when menuconfig is loaded I run:
`zcat /proc/config.gz > .cache/cachyos-km/pkgbuilds/linux-cachyos/src/linux-6.11/.config`
and load the config then save.
Which works fine but requires some extra work.

**Comment by 1Naim** [comment]
https://github.com/CachyOS/linux-cachyos/blob/master/linux-cachyos/PKGBUILD#L62

**Comment by shelterx** [comment]
@1Naim  Yes, I know but in the kernel manager GUI. PKGBUILDS gets overwritten.

**Comment by vnepogodin** [comment]
> We have "Use modpobed-db" as an option which is nice but I would really like to see "Use the running kernel config" or something similar, so it loads /proc/config.gz. I guess there's an issue with implementing something like this because it will override every setting in KM but as long as your clear with that, it should work.
> 
> The way I do it now is I enable menuconfig, start the build, then when menuconfig is loaded I run: `zcat /proc/config.gz > .cache/cachyos-km/pkgbuilds/linux-cachyos/src/linux-6.11/.config` and load the config then save. Which works fine but requires some extra work.

mhm I think adding option to modify _use_current, should be enough

**Comment by vnepogodin** [comment]
should be implemented in https://github.com/CachyOS/kernel-manager/commit/1df58cb5105ad7c0ae89f9421b7726557b9b39fd

**Comment by vnepogodin** [comment]
@shelterx is anything working as expected in latest version?

**Comment by shelterx** [comment]
I totally forgot about this. Thanks for fixing.
